test/parallel/test-repl.js

```diff
@@ -163,13 +163,23 @@ const errorTests = [
     send: '.break',
     expect: ''
   },
-  // Template expressions can cross lines
+  // Template expressions
   {
     send: '`io.js ${"1.0"',
```

Comment:
I guess recovering from e.g.

```js
'`abc ${ test'
```

is not possible anymore?

Reply:
Yes, only some forms are not possible. For instance, below one is possible.
```js
`abc ${
   test}`
```

Its mostly consistent with chrome console behaviour except for couple of cases.

1. we handle 'missing ) after argument list' error **better** than chrome console.
2. we allow repl commands in multiline mode which should otherwise be treated as properties/functions.
	 https://github.com/princejwesley/node/commit/c1796b8c7dc7e61b1d55177d8bab6a3a28b900bd (wip)

```
> (function() {
... x = { help: () => 'help received' };
... return x
...         .help ();    <-- no repl command parsing in multiline mode
.break    Sometimes you get stuck, this gets you out
.clear    Alias for .break
.editor   Enter editor mode
.exit     Exit the repl
.help     Print this help message
.load     Load JS from a file into the REPL session
.save     Save all evaluated commands in this REPL session to a file
...

```
(I'll give PR over the weekend)